.github/workflows/python-package.yml

```
@@ -32,63 +34,29 @@ jobs:
-          flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
-        shell: bash
-      - name: Run tests [macOS]
-        if: runner.os == 'macOS-latest'
-        run: |
-          python3 -m unittest discover -s tests -p "*_test.py"
-        shell: bash
-      - name: Install dependencies [Windows]
-        if: runner.os == 'Windows'
+          cache: 'pip'
+      - name: Cache pip dependencies
+        uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)